### DISCUSSION TITLE
Ubuntu 24.04 LXC  template has issues.

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#2904] [Announcements]
https://forum.proxmox.com/threads/ubuntu-24-04-lxc-containers-fail-to-boot-after-upgrade.145848/

https://lists.proxmox.com/pipermail/pve-devel/2024-April/063766.html

**Comment by tteck** [comment]
You can create a new Ubuntu 22.04 LXC and upgrade to 24.04

```
sed -i 's/jammy/noble/g' /etc/apt/sources.list
apt-get update
DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confold" dist-upgrade -y
rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
```

**Reply by rcv11x** [reply]
> You can create a new Ubuntu 22.04 LXC and upgrade to 24.04
> 
> ```
> sed -i 's/jammy/noble/g' /etc/apt/sources.list
> apt-get update
> DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confold" dist-upgrade -y
> rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
> ```

this is to upgrade my LXC jellyfin from version 22.04 to 24.04 (noble) i just tried it and i followed these instructions and there are many media especially HDR HEVC with tone mapping that before did not play in linux and now if! i hope i did well, greetings and thanks 

pd: the transcoding works perfectly, nothing has been changed, I check it on the proxmox host using intel_gpu_top and it works fine but on the lxc when I run the intel_gpu_top it gives error:

```
intel_gpu_top: ../tools/intel_gpu_top.c:557: get_num_gts: Assertion `!errno || errno == ENOENT' failed.
Aborted
```

**Reply by chrisbenincasa** [reply]
Hey - just wondering if you ever found a solution for the intel_gpu_top issue - I'm facing the same issue after upgrading my machine to Ubuntu 24

**Comment by tteck** [comment]
Looks as if the Ubuntu 24.04 template got pulled.

**Comment by tteck** [comment]
The Ubuntu 24.04 LXC template is now working.

**Comment by ghost** [comment]
Hmm, not working for me

 ✓ Updated LXC Template List
 |unable to create CT 123 - unsupported Ubuntu version '24.04'
bash: line 182: exit: A problem occured while trying to create container.: numeric argument required

**Reply by tteck** [reply]
The `6.8.4-3-pve` (or newer) kernel is needed to use the Ubuntu 24.04 LXC template.

**Reply by tteck** [reply]
What is the output of the command `pveversion` when run within the Proxmox shell?

**Reply by ghost** [reply]
That would be the exact problem! 
Update  + reboot and the script works as expected

Thank you for all of your hard work btw. Your scripts are amazing!!

**Comment by Petbotson** [comment]
Anybody else having trouble configuring a static ip with the image?

**Reply by tteck** [reply]
I don't see an issue. This is the LXC template, not the Ubuntu VM
```
   __  ____                __       
  / / / / /_  __  ______  / /___  __
 / / / / __ \/ / / / __ \/ __/ / / /
/ /_/ / /_/ / /_/ / / / / /_/ /_/ / 
\____/_.___/\__,_/_/ /_/\__/\__,_/  
 
Using Advanced Settings
Using Distribution: ubuntu
Using ubuntu Version: 24.04
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 110
Using Hostname: ubuntu
Using Disk Size: 2
Allocated Cores: 1
Allocated RAM: 512
Using Bridge: vmbr0
Using IP Address: 192.168.86.250/24
Using Gateway IP Address: 192.168.86.1
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Ubuntu LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 110 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.113.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```
